scipy/stats/tests/test_crosstab.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from scipy.stats import crosstab
+
+
+def test_crosstab_basic():
+    a = [0, 0, 9, 9, 0, 0, 9]
+    b = [1, 2, 3, 1, 2, 3, 3]
+    expected_avals = [0, 9]
+    expected_bvals = [1, 2, 3]
+    expected_count = np.array([[1, 2, 1],
+                               [1, 0, 2]])
+    (avals, bvals), count = crosstab(a, b)
+    assert_array_equal(avals, expected_avals)
+    assert_array_equal(bvals, expected_bvals)
+    assert_array_equal(count, expected_count)
+
+
+def test_crosstab_basic_1d():
```

Comment:
General question, given the similarity with `pandas` is it worthwhile to have a test a common input/output pair of the `crosstab` from this PR against what is in `pandas`?

Reply:
I don't really see the need.  We wouldn't actually use pandas in the test, so it would just be another result in a test case.

Reply:
yeah agreed, resolving.